src/Shared/PackageManagers/BaseProjectManager.cs

```diff
@@ -449,7 +458,7 @@ public virtual List<Version> GetVersions(JsonDocument? metadata)
         protected static readonly MemoryCache DataCache = new(
             new MemoryCacheOptions
             {
-                SizeLimit = 1024 * 1024 * 8
+                SizeLimit = 1024 * 1024 * 800
```

Comment:
Is the intent here to raise the size limit from 8 MB to 800 MB? Our Docker containers have finite memory allocations, typically only around 800 MB total - so 800 MB for just cache would be a bit much.

How about 100 MB instead?

```suggestion
                SizeLimit = 1024 * 1024 * 100, // 100 MB
```

Reply:
Addressed in latest revision